_i18n/en.yml

```diff
@@ -501,6 +501,15 @@ faq:
   avuln: >
     The Monero community has created a series of videos called "Breaking Monero", where potential Monero vulnerabilities are explored and discussed. There are 14 videos, with each exploring a different subject. Check out <a href="https://www.monerooutreach.org/breaking-monero/">the videos on monerooutreach.org</a>.
   vulnspotify: Available on Spotify as podcast
+  qcriminals: Is Monero for criminals?
+  acriminals: No. Monero is permissionless, peer-to-peer and decentralized. This means Monero developers and community have no control over how is used and who uses the technology. Monero is a tool in the hands of the public, very much like a kitchen knife or the US dollar, both of which are currently used to commit or fund horrible crimes, like terrorist attacks.
+  acriminals1: >
+    Monero, like every other currency, is money and can be used with good intentions (like funding charities) or bad intentions (like criminal activities). We cannot decide who uses Monero, but that doesn't mean we support criminal activities. Exactly like the Dollar is not considered 'for criminals', even if it's the preferred currency of terrorists, drug cartels and countless other criminal organizations.
+  qbanmonero: If you cannot avoid criminals using Monero, shouldn't it be banned?
+  abanmonero: Banning Monero would be a minor annoyance for criminals, but a major problem for those who use this currency to escape war, mass surveillance or sexual/political discrimination by tyrannical regimes or organizations.
+  abanmonero1: Banning Monero is impossible by design, but even if it was possible, the only consequence of banning it would be the sudden rise of dozens of Monero clones. Exactly how it happened with Silk Road, the pioneer darknet market. Closing it resulted in countless more darknet markets being created, many of which didn't share the ethos of the original Silk Road
```

Comment:
```suggestion
```

Reply:
I would remove this whole paragraph. Silk Road was a crime and Monero is not. Shouldn't be placed in the same category.

Reply:
How is that related to my point? I'm saying that closing a darknet market will only result in many more being opened.

Reply:
If people are afraid of mentioning Silk Road for fear of association (i absolutely disagree with this), i guess we can avoid naming it or talk about it indirectly, but of course the impact of this paragraph on the reader will be heavily reduced.